keyboards/custommk/genesis/rev1/keymaps/default/keymap.c

```diff
@@ -18,14 +18,14 @@
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
-	[0] = LAYOUT_ortho_5x4(
+	[0] = LAYOUT_ortho(
```

Comment:
This sholud be reverted.  It appears to fit the layouts.

Reply:
Naming has been reverted